mediapipe/tasks/ios/text/text_classifier/sources/MPPTextClassifier.h

```diff
@@ -0,0 +1,102 @@
+ * Metadata](https://www.tensorflow.org/lite/convert/metadata")that contains the mandatory
+ * (described below) input tensors, output tensor, and the optional (but recommended) label items as
+ * AssociatedFiles with type TENSOR_AXIS_LABELS per output classification tensor.
+ *
+ * Metadata is required for models with int32 input tensors because it contains the input process
+ * unit for the model's Tokenizer. No metadata is required for models with string input tensors.
+ *
+ * Input tensors
+ *  - Three input tensors `kTfLiteInt32` of shape `[batch_size xbert_max_seq_len]`
+ *    representing the input ids, mask ids, and segment ids. This input signature requires a
+ *    Bert Tokenizer process unit in the model metadata.
+ *  - Or one input tensor `kTfLiteInt32` of shape `[batch_size xmax_seq_len]` representing
+ *    the input ids. This input signature requires a Regex Tokenizer process unit in the
+ *    model metadata.
+ *  - Or one input tensor ({@code kTfLiteString}) that is shapeless or has shape `[1]` containing
+ * the input string.
+ *
+ * At least one output tensor `(kTfLiteFloat32}/kBool)` with:
+ *  - `N` classes and shape `[1 x N]`
+ *  - optional (but recommended) label map(s) as AssociatedFile-s with type TENSOR_AXIS_LABELS,
+ *    containing one label per line. The first such AssociatedFile (if any) is used to fill the
+ *    `class_name` field of the results. The `display_name` field is filled from the AssociatedFile
+ *    (if any) whose locale matches the `display_names_locale` field of the
+ * `MPPTextClassifierOptions` used at creation time ("en" by default, i.e. English). If none of
+ * these are available, only the `index` field of the results will be filled.
+ *
+ * @brief Performs classification on text.
+ */
+NS_SWIFT_NAME(TextClassifier)
+@interface MPPTextClassifier : NSObject
+
+/**
+ * Creates a new instance of `MPPTextClassifier` from an absolute path to a TensorFlow Lite model
+ * file stored locally on the device and the default `MPPTextClassifierOptions`.
+ *
+ * @param modelPath An absolute path to a TensorFlow Lite model file stored locally on the device.
+ *
+ * @param error An optional error parameter populated when there is an error in initializing
+ * the text classifier.
+ *
+ * @return A new instance of `MPPTextClassifier` with the given model path. `nil` if there is an
+ * error in initializing the text classifier.
+ */
+- (instancetype)initWithModelPath:(NSString *)modelPath error:(NSError **)error;
+
+/**
+ * Creates a new instance of `MPPTextClassifier` from the given `MPPTextClassifierOptions`.
+ *
+ * @param options The options of type `MPPTextClassifierOptions` to use for configuring the
+ * `MPPTextClassifier`.
+ *
+ * @param error An optional error parameter populated when there is an error in initializing
+ * the text classifier.
+ *
+ * @return A new instance of `MPPTextClassifier` with the given options. `nil` if there is an error
+ * in initializing the text classifier.
```

Comment:
```suggestion
 * @param options The options of type `MPPTextClassifierOptions` to use for configuring the
 *     `MPPTextClassifier`.
 * @param error An optional error parameter populated when there is an error in initializing
 *     the text classifier.
 * @return A new instance of `MPPTextClassifier` with the given options. `nil` if there is an error
 *     in initializing the text classifier.
```

I have never seen this spacing before so I am not sure whether this is correct or not. Treat this as my gut feeling...

Reply:
It was inconsistent. Removed spacing between params. but I have retained the spacing between @param block and @return.
Following the style in the tensorflow repo.[ An example](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/objc/apis/TFLInterpreter.h#L128)